css-pseudo-classes/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <link rel="stylesheet" href="styles.css"/>
+  <title>css-pseudo-classes</title>
+</head>
+<body>
+  <h1>Hover Button</h1>
+<div class="container">
+  <button id="btn">I'm a button!</button>
+</div>
+<h2>Striped Table</h2>
+  <table>
```

Comment:
I would be sure to include the `thead` and `tbody` elements in your tables! It is better for indexing, and carries important semantic information for the table!